compendium/test_bundles/TestBundleDSFAC1/src/ServiceImpl.cpp

```diff
@@ -0,0 +1,30 @@
+#include "ServiceImpl.hpp"
+#include <iostream>
+
+namespace sample
+{
+
+    ServiceAImpl::ServiceAImpl(std::shared_ptr<cppmicroservices::AnyMap> const& props,
+                                               const std::shared_ptr<test::ServiceBInt> interface1)
+        : properties(props)
+        {
+        if (nullptr != interface1)
+        {
+          
+        }
```

Comment:
this code does nothing. is the if statement supposed to do something? If not, please delete this.

Reply:
Will fix